src/app/Mappings/components/MappingBuilder/helpers.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  MappingSource,
+  Mapping,
+  MappingItem,
+  MappingType,
+  MappingTarget,
+  INetworkMappingItem,
+  IStorageMappingItem,
+} from '../../types';
+import { IMappingBuilderItem } from './MappingBuilder';
+import { IVMwareProvider, ICNVProvider } from '@app/Providers/types';
+
+export const getSourceById = (sources: MappingSource[], id: string): MappingSource | undefined =>
```

Comment:
Is this ever called?

Reply:
It's called in `getBuilderItemsFromMapping`.